src/3d/qgsfeature3dhandler_p.h

```diff
@@ -45,8 +45,9 @@ class Qgs3DMapSettings;
  *
  * \note Not available in Python bindings
  */
-class Qgs3DRenderContext
+class Qgs3DRenderContext : public QObject
```

Comment:
Since the signal for cancellation is gone now, we probably don't need Qgs3DRendererContext to be derived from QObject anymore (and Q_OBJECT here and in QgsPointCloud3DRenderContext